### PR TITLE
Implemented modal window 'activate/deactivate' for checks and services

### PR DESCRIPTION
**Pull request by lisovyk**
Added module window for activate/deactivate

### CODE REVIEW

File: moninag/static/src/ts/app/checks/activate-deactivate.component.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  template: require('./activate-deactivate-window.html'),
+})
+
+export class DialogResultDialog{
+    checkstate: boolean;
+    actionName: string;
+
+    constructor(public dialogRef: MdDialogRef<DialogResultDialog>) {
+    }
+}
```
Comment (lhalam) [inline moninag/static/src/ts/app/checks/activate-deactivate.component.ts]: add endline

File: moninag/static/src/ts/app/checks/activate-deactivate-window.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div>
+    <h1 md-dialog-title>Attention!</h1>
+    <div md-dialog-content>Are you sure you want to {{actionName}} ?</div>
+    <div md-dialog-actions>
+        <button *ngIf="!checkstate" md-button (click)="dialogRef.close(true)">Activate</button>
+        <button *ngIf="checkstate" md-button (click)="dialogRef.close(false)">Deactivate</button>
+        <button  md-button (click)="dialogRef.close()">Cancel</button>
+    </div>
+</div>
```
Comment (ghost) [inline moninag/static/src/ts/app/checks/activate-deactivate-window.html]: endline

File: moninag/static/src/ts/app/checks/activate-deactivate.component.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { CheckDetailComponent } from './check-detail.component';
+import { ChecksService } from './checks.service';
+
+@Component({
+  selector: 'dialog-result-dialog',
+  template: require('./activate-deactivate-window.html'),
+})
+
+export class DialogResultDialog{
+    checkstate: boolean;
```
Comment (ghost) [inline moninag/static/src/ts/app/checks/activate-deactivate.component.ts]: camelCase property name

File: moninag/static/src/ts/app/checks/activate-deactivate.component.ts (hunk 1 above)
Comment (lisovyk) [reply moninag/static/src/ts/app/checks/activate-deactivate.component.ts]: done!

**Review by ghost — COMMENTED** [review]
Squash commits. Commit should contain part of code that works and is valid (code style and other stuff). So there is not need to have the fix commit after not properly edited code commit (fixup it).

**Review by randombluff — APPROVED** [review]
;)

**Review by nkoblai — APPROVED** [review]

**Pull request merged**